.github/workflows/wheel.yml

```
@@ -0,0 +1,407 @@
+            python: cp313-macosx_arm64
+            arch: arm64
+
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        
+      - uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3 # v5.2.0
+        name: Install Python
+        # https://cibuildwheel.pypa.io/en/stable/faq/#macos-building-cpython-38-wheels-on-arm64
+        with:
+          python-version: '3.8'
+        
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@d4a2945fcc8d13f20a1b99d461b8e844d5fc6e23 # v2.21.1
+        env:
+          CIBW_BUILD: ${{ matrix.python }}
+          CIBW_ARCHS: ${{ matrix.arch }}
+
+      - uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
+        with:
+          name: cibw-wheels-${{ matrix.python }}
+          path: ./wheelhouse/*.whl
+
+  # ---------------------------------------------------------------------------
+  # Windows Wheels
+  # ---------------------------------------------------------------------------
+
+  windows:
+    name: Build wheels on Windows
+    runs-on: windows-2022
+    if: |
+      github.event_name != 'schedule' ||
+      github.repository == 'AcademySoftwareFoundation/OpenImageIO'
+    strategy:
+      matrix:
+        include:
+          # -------------------------------------------------------------------
+          # CPython 64 bits
+          # -------------------------------------------------------------------
+          - build: CPython 3.8 64 bits
+            python: cp38-win_amd64
+            arch: AMD64
+          - build: CPython 3.9 64 bits
+            python: cp39-win_amd64
+            arch: AMD64
+          - build: CPython 3.10 64 bits
+            python: cp310-win_amd64
+            arch: AMD64
+          - build: CPython 3.11 64 bits
+            python: cp311-win_amd64
+            arch: AMD64
+          - build: CPython 3.12 64 bits
+            python: cp312-win_amd64
+            arch: AMD64
+          - build: CPython 3.13 64 bits
+            python: cp313-win_amd64
+            arch: AMD64
+
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3 # v5.2.0
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@d4a2945fcc8d13f20a1b99d461b8e844d5fc6e23 # v2.21.1
+        env:
+          CIBW_BUILD: ${{ matrix.python }}
+          CIBW_ARCHS: ${{ matrix.arch }}
+
+      - uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
+        with:
+          name: cibw-wheels-${{ matrix.python }}
+          path: ./wheelhouse/*.whl
+
+
+  upload_pypi:
+    needs: [sdist, linux, linux-arm, macos, macos-arm, windows]
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v3')
```

```suggestion
    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v3.0.')
```
I think this change will ensure that we only upload to PyPI for tagged releases in the 3.0 branch, which are the supported releases. We can modify this in the future as new branches become supported, but for now, tags that look like v3.1 are just developer releases and so should not go to PyPI. Does that sound right ot everybody?